### ISSUE TITLE
Unknown Track Metadata

### ISSUE THREAD

**Issue by alexlewer** [ebaauw/homebridge-zp#44]
Hi Erik,

Just noticed an error popping up in my Homebridge logs over the past few days regarding _unknown track metadata_. Doesn't seem to be causing any issues, but might be worth a look when you get a sec - relevant console log lines below:

```
[2018-8-28 10:47:34] [ZP] Initializing ZP platform...
[2018-8-28 10:47:34] [ZP] homebridge-zp v0.2.31, node v9.11.2, homebridge v0.4.44
[2018-8-28 10:47:34] [ZP] warning: not using recommended node version v8.11.4 LTS
[2018-8-28 10:48:04] [ZP] Initializing platform accessory 'Bedroom Sonos'...
[2018-8-28 10:48:05] [ZP] Bedroom Sonos: coordinator for group Bedroom
[2018-8-28 10:48:05] [ZP] Bedroom Sonos: unknown track metadata {"$":{"id":"-1","parentID":"-1","restricted":"true"},"res":[{"_":"x-sonos-http:librarytrack%3ai.xYB7oi8YEdP.mp4?sid=204&flags=73760&sn=1","$":{"protocolInfo":"sonos.com-http:*:audio/mp4:*","duration":"0:03:14"}}],"r:streamContent":[""],"r:radioShowMd":[""],"upnp:albumArtURI":["/getaa?s=1&u=x-sonos-http%3alibrarytrack%253ai.xYB7oi8YEdP.mp4%3fsid%3d204%26flags%3d73760%26sn%3d1"],"dc:title":["Make Me Feel"],"upnp:class":["object.item.audioItem.musicTrack"],"dc:creator":["Janelle Monáe"],"upnp:album":["Dirty Computer"],"r:tags":["1"]}
[2018-8-28 10:48:05] [ZP] Bedroom Sonos: set power (play/pause) from undefined to false
[2018-8-28 10:48:05] [ZP] Bedroom Sonos: set current track from undefined to "Make Me Feel"
[2018-8-28 10:48:05] [ZP] Bedroom Sonos: set group volume from undefined to 9
[2018-8-28 10:48:05] [ZP] Bedroom Sonos: set group mute from undefined to false
```

Alex

**Comment by ebaauw** [comment]
What audio source were you playing?

**Comment by alexlewer** [comment]
Nothing at that point in time; that log occured when I restarted homebridge and the Sonos was not playing anything. I think potentially that was from when I previously tried to stream a file from my local iPhone library.

**Comment by ebaauw** [comment]
Hm, it shows something else when I play a song from my iPhone, but I'll add this one anyways.  The warning can indeed be ignored.

**Comment by ebaauw** [comment]
Fixed in v0.2.32.

**Comment by alexlewer** [comment]
Confirmed; fixed in v0.2.32.
